src/configuration.js

```diff
@@ -143,6 +144,10 @@ export default class Configuration {
       if (config.reporter.agentPort) {
         senderConfig['port'] = config.reporter.agentPort;
       }
+
+      if (config.reporter.timeout) {
```

Comment:
timeout or timeoutMs? Is there a test?

Reply:
Good catch. Only `configuration_env.js` was tested. Added a test for `configuration.js` [here](https://github.com/jaegertracing/jaeger-client-node/pull/350/files#diff-0563bbe36f9d317234592eb362aa4115R196).